litmus/1.0.0/README.md

```diff
@@ -0,0 +1,52 @@
+# Litmus Chaos Keptn Integration
+
+![](https://raw.githubusercontent.com/keptn-sandbox/litmus-service/master/assets/litmus-keptn.png)
+
+Use Litmus Chaos to perform chaos tests on your applications triggered by Keptn using the LitmusChaos framework.
+
+This integration uses the [Job Executor Service](https://artifacthub.io/packages/keptn/keptn-integrations/job-executor-service) before continuing, ensure you have installed and configured the Job Executor Service.
+
+The instructions below assume you will configure the [Job Executor Service](https://artifacthub.io/packages/keptn/keptn-integrations/job-executor-service) to listen for `sh.keptn.event.chaos.triggered` events. Feel free to change `chaos` to whatever your shipyard task is called. If you do so, also modify the given `job/config.yaml` file below.
+
+## Use Litmus Chaos with Job Executor Service
+
+1. Change to the relevant stage branch in the Keptn Git upstream repo eg. `dev`
+1. Create two folders: `job` and `locust`
+1. Create a file: `job/config.yaml` with the contents below
+1. Upload your locust specific files inside the `locust` folder
+
+
+### **job/config.yaml**
+```
+apiVersion: v2
+actions:
+  - name: "Print files"
+    events:
+      - name: "sh.keptn.event.chaos.triggered"
+    tasks:
+      - name: "Run locust tests"
+        files:
+          - locust/
+        image: "locustio/locust"
```

Comment:
why does it say locust here, but in the title you are trying to use litmus? I think this is the wrong job/config.yaml :D